app/components/ui/HamburgerMenu.module.scss

```diff
@@ -9,4 +9,12 @@
     background: #276ce5;
     color: #ffffff;
   }
+
+  &.emphasized {
+    font-weight: 600;
+  }
+
+  &.hidden {
+    display: none;
+  }
```

Comment:
This felt like a weird way to feature flag the button, but conditionally adding it to the links array wasn't working out. Was trying to concat it onto links if `showReportCrisis === true`, but it wasn't showing up. I haven't done react in a bit--is that something that needs to run in a lifecycle hook or something?

Reply:
Were you literally using the [`Array.concat()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat) method? That method doesn't mutate the original array, and instead returns a new one, so it'd have to be done in the same expression as the one where you assign to `links`.

If that doesn't work, then we're admittedly doing some weird stuff in JavaScript-land by having lots of top-level/import-time logic for the whitelabel, and I wonder if that's causing the issue you're seeing. If you're still hitting issues after making sure that you are in fact mutating/initializing the `links` array properly, then I wonder if moving the `links` array definition into the body of the HamburgerMenu component would avoid the import-time issues.

Failing both of those, then I think what you have now is probably fine for getting this feature out the door, but it'd be nice to create a ticket to investigate the issue further.

Reply:
Ahh dangit I forgot that concat is non-destructive. Let me try that again.